mobile/components/views/User/UserDashboard.tsx

```diff
@@ -31,11 +31,17 @@ const UserDashboard = ({ navigation }: any) => {
/VRajh0pfnwFh1WSS8SMu1s3MdLm6cCNQPyWxb2ocdGyeJiB6r0tntI3VwXTGxoduA2XEIqT3SYZs85duTyuMGY4TuY1jh+izcMuTa1oGgO+q13YtSDZLt/X04DxXL4lXz1pB3PAz813k2qSeN89eBaRZMilDJ0a7nX4hdNZUFRpID4JIOmbqPxVirigyaQZjjEeawKdWaYDtRHio21QQWg8dj811+qknce/yIP00Wlfb6HveGXZogndV1PPmJ+SgtHQ53kxu+qdGX6QxLHk47ghNKFeKBEpKaIQB7oQhAAhNCABWba4ytA4OqCfISqq9Aycp5OM+iqa2TjjtFzRJPLz5MyMbqS53MkkdYXz/Fa/eK7TtYHe0AaeBjzXDYnZ1RJLdOazdJGn4meguoWjtf4Z9nmuaLytq1kmmP9zx8R6D6+C6jE+07JIcDA0nZZWG1jQtqVIHRtKnPU5RPzlUcUu6LtHCCf3KT1TlxG+vPH50IYadzneRX5ewnc45Sf/3MvQkrKv3tAkBzmn421AJHSVUr4VPea6QN45cVlXNMs3zN5AggDz4lEMcZStSLjxxj0LdtD61NlJ78xq0+66CJzDivptWm4QHa+a4f+HVmKlyajiSKbCR3dMzu6PlK+j3GvBWlBOzP1WW5pUUqFEF23Dn0XrdbnpHDkrFOATt+qpXNUKGTUEcxucipWqkKpUxQAFe9R4K57tBaPaz2jRMbjoqqyOckky1HHFdSFfEjmgHSCVWGIT8X6rnWYh3teWq9jU5Kx+nS4J8ThJnV22IaaFH+NlhJc9uWNi2Sek8AuRFxHFSq3EjdcrTK7JnCDN2+xAHWd9dNQfNVLOuQ8GdJ4nQLKpvXo24JIAEDx1KlWJJUSxklGj6GLhmVvfERwj8FV9puQdCVmWDTEH8lZNbWGt5aQs+UeaKS8PBs4UMx1RC9MOouyyG9Uq3vH1Wn6M8Np9+TH9Jq6l5cEEJhJaxkiKE0kCLCcKSSAIoKZSQALQt6Qa0ZtyZVvDMOhntHDU7A8BzXrVgDvHTmqufxKi5p4077mFjuH0oNRwkn5QufrBopEkSNVd7T4mHQ1p0Ex1KwsTustIDpCxpxUsnh8/8AvzPRaWMtiT6m1iRAZI2yiI8NFxNapLiVs0bwvtm6/CAfEaH6Lna5gqTS49tpk+GW1NFlt25uxMAz5qVxiheIcdNdx++QVBlQqL6gJAAkkgT8tla9Wm7oeSUXGz6P/DqyyUHVcse0cY8BoFv164zQOUqjhVYU6DWjYNHDoql1iIExE/v81x66Kx358mVODllbLlxeRKwr7EgDqql5ioA5/qsWpXzGZVRY5ZHcjUwYElyb1PEm8SvG8xcEbyOR4rAq1uCz7tx4FSx0sW7Z1OMUrKOJuhxcNpPkqtK+I0kx9F53NVx0VAugrXhjTjTPOanVOOTdHhG0y7lerayq4RhlWvJaNG7nh4K6bF1P3tOSinsTq+S9p8mXJFSrjzH7U7KbKvL1XnTt3u2aY8N1uYTgbnEFwgSJJ/soZzjBWy5ByZ74bnfoBJ5/qFepVXMqDNwdouhwtlJoyMGuxKy8bpim/fis7epyquGW9O4Slto7nDXB9Bro4arNv6eshe3Zu7abcd4HTbeFBzw58eSuRzKChLuYmXA5PJApBBTc2CQeBI9EoWuYIkIKEAWkIQmMS9bKlmqAHaZPkvJXMKHfJ4BqjyOkd4VumkzcN0Jy8lz3aS9IGULSrOjvujwC5TGLrPU0J39fFZmqzNQ2939DY0eFOe7sjIq0s51WZixgQVrveBusLFq4JhU8NuS8jZxu5GVa4h7Mlh2mfVF1WadQVk4uIIcPNVW3RWosKfiRm5NWseRwfb6Gg6pqna1/9amP62j1MLO/mio0rqKjSeDmn0KkWJlfJrY9mfX6twW0x4Bcxf4jqRO4+vVe9fFwaTYOuVc/WrSSZWXp8L/9GnCrste0JTbUEQqbKw5odU4HQ9Va2k8sy8z1quHNUbiuNpVe4rqlUfKnhjM7UayuEe1evyWe8S6OZCk5QomHtP8AUPqrUIpdDD1OV5GlI+k9n+5TyNDYDGmY1M/sq9SsWuJLxmPMrn7W6LCHDXXUcxyW/b31MtkVAJ4HQ+Cws0JJ7l3PYOCSqK4LTcObyAGnBXjR7mUeuizrGqXnefoth9Wm1skEny/FVMm66ZxNbXyVsEoFjyDrv1+q8+0NEydJ5dV629yPaiCQC3UaR01XljlTjOy7jJtpvrf2HBv1yl7Dw7PVnMBHCVrWlUmpPVY2HvDQCWyDPHjH6yunwXDiRnOnLRSKMpz4INVmjFyk+5G9+0d4z6ifxXirGIiKjvL6BVl6OPRHkp8SfvBEJoTIiwgBNqaZIKFYtHwDzJ+QXgU6e/p9VX1Mmope0taONzfuPbFnksgctTK5C/a4TzHD812R2PiuYv8A7Sp9531KzNZjUWp/A2dHO/CcpdYm4HK4Edd1n3NZsSCZ+Xmr+M7FYZUmGKcbXBd/aVr6pIIKy7Gg+pUbTY0uc5wa0Dck7K3erpP4XMH887QaUqkabcNOS0cfETzutk55fcUe0XZG4tGl78rqcgZ2HQE7SDqFzVOkXuholfYv4n/9Od/5aX/svmeEnuLqctkbIdNj/U5FB8HhUqFhyl+wCgbjqo4z7/kqDkoQUopkmfUyxZJQXRPjlmmy5IMg7c07nEX1CS8ySZlZYTK69WiJ6ubX+ls1BzXi+qvAphNRRDLPJuiReSvWhTkqLFaoJSdIlw4t0rkzbtavdjkr1rErDtd1q2u6oZI0eq0mZyStHR4VXjkoYxdv2ndVrH/6H4Ivt1SUF6yyy+ZlnC6rg+M0aDeNlcxG4Bc1pI38dPELLH2rfutVq6+0b4LmUFuUvYNx5sv2WHup1ATqxzmHKNYk8ORHzX0iq9rWtYznuPX9+K4qrszxb+C6fDPh8fwVnTT5aowNYm0pN9DMv/tHfej0XgvW69933ivJay6Iw8j8bBJNCZwf/9k=',
```

Comment:
Should we put this base64 to repo?

Reply:
This wont be base 64, wthis will be link to avatar, just mocked it